CDCgov/prime-reportstream
# Monkeypox schema updates

This PR updates the monkeypox schemas

1. Sends a hash of the patient_id to DCIPHER
2. Fixes a hardcoded value sample_type_other_text for DCIPHER

Test Steps:
1. Submit a test monkeypox result to RS. Confirm DCIPHER Patient_id is a hash and changes per patient_id (PID-3); confirm sample_type_other_text is blank.

## Checklist

### Testing
